app/client/Last.fm Scrobbler.css

```diff
@@ -339,6 +340,10 @@ NothingPlayingWidget QAbstractButton#itunes {
 background-image: url(":/control_bar_scrobble_itunes.png");
 }
 
+NothingPlayingWidget QAbstractButton#applemusic {
+background-image: url(":/control_bar_scrobble_applemusic.png");
```

Comment:
[minor] Don't we need indentation here?

Reply:
I would choose indentation if this was a new file, but I think it makes more sense to follow the style already in the file.